Anguandia/quickcredit
# Ft db create loan 166114379

#### Purpose

Integrate database with loan creation

#### Tasks to be completed

- Add check for duplicate application in utility module availability
- Add query in create a function for inserting new loan instance into
database
- Disable authentication for till when authentication is integrated with
database

#### How to manually test this feature

- Clone this repo and run npm install to install this app locally
- Start the app with either npm start or debug=quickcredit npm run dev
commands in your terminal
- Proceed to create a loan with a POST/api/v1/loans request supplying a
json object with values for the fields of email(user), amount(float) and
tenor(integer)
- The response should return details of the created loan object
- Restart the application and run a GET/api/v1/loans request on postman
using localhost and port 3000
- You should see the previously created loan in the response

#### Pivotal tracker stories

[finishes #166114379]
